pkgs/top-level/agda-packages.nix

```diff
@@ -26,5 +26,8 @@ let
     cubical = callPackage ../development/libraries/agda/cubical { };
 
     generic = callPackage ../development/libraries/agda/generic { };
+
+    fla = callPackage
```

Comment:
Why not call it `functional-linear-algebra`?

Reply:
That makes more sense; I had been using `FLA.*` as imports inside the library because the name was too long but that doesn't really matter as much here since the name won't be part of a long identifier.